pandas/tests/extension/base/methods.py

```diff
@@ -116,10 +116,8 @@ def test_argsort_missing_array(self, data_missing_for_sorting):
         tm.assert_numpy_array_equal(result, expected)
 
     def test_argsort_missing(self, data_missing_for_sorting):
-        msg = "The behavior of Series.argsort in the presence of NA values"
-        with tm.assert_produces_warning(FutureWarning, match=msg):
-            result = pd.Series(data_missing_for_sorting).argsort()
-        expected = pd.Series(np.array([1, -1, 0], dtype=np.intp))
+        result = pd.Series(data_missing_for_sorting).argsort()
+        expected = pd.Series(np.array([2, 0, 1], dtype=np.intp))
```

Comment:
are NAs supposed to be sorted to the front or the end?  IIUC this is putting them at the front?

Reply:
I think this is correct as the NA should be put at the end according to the deprecation message.

The fixture says

```
    This should be three items [B, NA, A] with
    A < B and NA missing.
```

Reply:
Ok, I think I was confusing this with ‘rank’